pkg/logging/logging.go

```diff
@@ -2,21 +2,21 @@
 //
 // This package supports runtime configuration via the following
 // environment variables:
-//   * LOGLEVEL supporting the following values (case insensitive), order is indicative of importance:
-//     * fatal
-//     * panic
-//     * error
-//     * warn
-//     * info
-//     * debug
-//   * LOGENCODING supporting the following values:
-//     * json
-//     * console
-//   * MODULE_LOGLEVELS supporting ,-separated module=level pairs, e.g.: grpc=debug,kubernetes=warn
-//   * MAX_LOG_LINE_QUOTA in the format max/duration_in_seconds, e.g.: 100/10
-//   * PERSISTENT_LOG supporting the following values for additional log file on persistent storage
-//     * true
-//     * false
+//   - LOGLEVEL supporting the following values (case insensitive), order is indicative of importance:
+//   - fatal
+//   - panic
+//   - error
+//   - warn
+//   - info
+//   - debug
```

Comment:
It looks like there are multiple herarchy/indentation levels in that comment.
Any idea to ensure the hierarchy is still displayed/respected ?

Reply:
Nice catch, there is a solution in docs https://go.dev/doc/comment#:~:text=versa.%20For%20example%3A-,//%20Here%20is%20a%20list%3A,-//%0A//%20%201.%20Item%201